server/models/User.js

```diff
@@ -105,6 +105,11 @@ function defineModel() {
         type: DataTypes.ARRAY(DataTypes.STRING),
         allowNull: true,
       },
+      changelogViewDate: {
+        type: DataTypes.DATE,
+        defaultValue: DataTypes.NOW,
```

Comment:
Why a defaultValue of NOW?

Reply:
Yes, this is also related to, https://github.com/opencollective/opencollective-api/pull/6147#discussion_r651190923. I've corrected it. 😯